Game.js

```diff
@@ -63,12 +55,13 @@
 
         // startup the ticker
         createjs.Ticker.setFPS(frameRate);
-        createjs.Ticker.addEventListener("tick", gameController.onTick);
+        createjs.Ticker.addEventListener("tick", gameController.onTick.bind(this, stage));

```

Comment:
This is a weird line. If you're not familiar, `.bind()` changes the context of a function call so that you can change the context of `this` and inject new params into function calls you might not be able to otherwise.

This is NOT a good way of doing it, but I did it for example and as a quick debugging fix on my end.